### DISCUSSION TITLE
AntSDR IP clarification

### DISCUSSION THREAD

**Discussion by alphafox02** [srsran/zynq_timestamping#21] [General] [answered]
Finally getting around to using my AntSDR w/ srsRAN. I noticed this page,

https://github.com/srsran/zynq_timestamping/tree/main/projects/antsdr

mentions one IP starting with 10.x

However, the application note mentions exporting the default 192.168.1.10 IP but then goes on to mention that the ue conf file in the AntSDR needs to be the 10.x IP. 

Just looking to clarify what's needed, currently I have the firmware/SD card loaded with the BIN file and can ping and use the AntSDR at 192.168.1.10. Should I be going in via console and making changes?

**Comment by sauka** [comment, marked as answer]
Just use whatever IP address suitable for you, e.g. the default one 192.168.1.10. `make load-boot` mentioned in the README will use this address. And then use this same address for BOARD_IP variable that will be propagated to ue.conf.

In the appnote we just show how to set a custom IP if needed and manually transfer some bootfiles.

**Comment by alphafox02** [comment]
That makes sense, may be easier to stick with the default on so I don't have to go in and change eth0. I almost have it working, however some issues with the AntSDR making a connection to the ENB. I'll detail that separately.
